server/camcops_server/cc_modules/cc_forms.py

```diff
@@ -983,20 +984,20 @@ def after_bind(self, node: SchemaNode, kw: Dict[str, Any]) -> None:
         self.description = _("Which ID number to link on?")
 
 
-class OptionalWhichIdNumSelector(MandatoryWhichIdNumSelector):
-    """
-    Node to select (optionally) an ID number type.
-    """
-    default = None
-    missing = None
-
-    def __init__(self, *args, **kwargs) -> None:
-        self.allow_none = True
-        super().__init__(*args, **kwargs)
-
-    @staticmethod
-    def schema_type() -> SchemaType:
-        return AllowNoneType(Integer())
+# class OptionalWhichIdNumSelector(MandatoryWhichIdNumSelector):
```

Comment:
Do we need this class any more? Can we remove it altogether?

Reply:
Can remove!